adcm_client/packer/naming_rules.py

```diff
@@ -86,12 +86,18 @@ def resolve_build_id(git_data, master_branches, timestamp):
                 build_id = '-1'
             else:
                 build_id = '-' + git_data['branch'].replace("-", "_").replace("/", "_")
+            if timestamp_enabled:
+                build_id = build_id + '_' + timestamp
     else:
         build_id = '-1'
+        # timestamp
+        if timestamp_enabled:
+            build_id = build_id + '_' + timestamp
+
```

Comment:
I don't believe that users locally reproduce pr state, even so this is special state and he must understand why it is needed, so he wont use `-T` option.

Even it this situation may occur `-T` is just ignored in this code so we may log an warning in this case.
Any way this case may be checked separately so code will become more easy to read. 

```suggestion
    else:
        build_id = '-1'
        # timestamp
    if timestamp_enabled:
        build_id = build_id + '_' + timestamp

```